Medellín, Colombia 24 de Abril de 2026
<table style="border: none; border-collapse: collapse; width: 100%;">
    <tr style="border: none;">
        <td style="border: none; width: 120px; vertical-align: middle;">
            <img src="img/logo.png" style="width: 100px;"/>
        </td>
        <td style="border: none; vertical-align: middle;">
            <div style="color:#BA352C; font-size: 30px; font-weight: bold; line-height: 1.2;">
                Universidad Autónoma Latinoamericana
            </div>
            <div style="color:#FC9C28; font-size: 24px; font-weight: bold; line-height: 1.2;">
                Especialización, Analítica de datos - Grupo 2
            </div>
        </td>
    </tr>
</table>

<span style="font-size: 18px;"> Entregable Final de la Asignatura: Principios analíticos para la ciencia de datos </span>

### Integrantes 

- Evelin Dahiana Franco Campiño
- Kevin Mateo Jiménez Guisao
- Nicolle Gaviria Mejía

</br>

***


# Análisis Exploratorio de Datos (EDA)
## Salud mental estudiantil y síndrome de burnout

### 📄Introducción

<p>
    El Conjunto de Datos sobre Salud Mental y Agotamiento Estudiantil (150k filas, 20 características) es una base de datos a gran escala diseñada para analizar y predecir los niveles de burnout (agotamiento) estudiantil basándose en factores académicos, psicológicos y de estilo de vida. 
</p>

    Este conjunto de datos contiene 150,000 registros de estudiantes con una combinación de variables numéricas y categóricas, lo que lo hace ideal para tareas de aprendizaje automático (machine learning), clasificación y análisis de datos.


### 🧮Conjunto de Datos
**Caracteristicas:**
- Demografía: Edad, género, carrera y año académico.

- Indicadores académicos: Promedio general (CGPA), asistencia y horas de estudio.

- Puntajes psicológicos: Niveles de ansiedad, depresión y estrés.

- Factores de estilo de vida: Horas de sueño, actividad física y tiempo frente a la pantalla.

- Indicadores de estrés social y financiero.

- Variable objetivo: `burnout_level` (nivel de agotamiento).

<table style="width: 500px; margin-left: 0; margin-right: auto;">
    <thead>
        <tr style="background-color: #BA352C; color: white;">
            <th style="padding: 10px; text-align: left; border: 1px solid #ddd;">Información</th>
            <th style="padding: 10px; text-align: left; border: 1px solid #ddd;">Detalle</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td style="background-color: #FC9C2833; font-weight: bold; padding: 8px; border: 1px solid #ddd; width: 150px;">Fuente</td>
            <td style="padding: 8px; border: 1px solid #ddd; ; width: 350px;">
                <a href="https://www.kaggle.com/datasets/sehaj1104/student-mental-health-and-burnout-dataset/data" target="_blank">Kaggle - Dataset Link</a>
            </td>
        </tr>
        <tr>
            <td style="background-color: #FC9C2833; font-weight: bold; padding: 8px; border: 1px solid #ddd; width: 150px;">Nombre del dataset</td>
            <td style="padding: 8px; border: 1px solid #ddd; width: 350px;">Student Mental Health and Burnout Dataset</td>
        </tr>
        <tr>
            <td style="background-color: #FC9C2833; font-weight: bold; padding: 8px; border: 1px solid #ddd; width: 150px;">Propietario</td>
            <td style="padding: 8px; border: 1px solid #ddd; width: 350px;">Mansehaj Preet</td>
        </tr>
        <tr>
            <td style="background-color: #FC9C2833; font-weight: bold; padding: 8px; border: 1px solid #ddd; width: 150px;">Tipo de Origen</td>
            <td style="padding: 8px; border: 1px solid #ddd; width: 350px;">Archivo CSV</td>
        </tr>
    </tbody>
</table>

#### Importacion del conjunto de datos

In [ ]:
from IPython.core.display import Markdown
from IPython.core.display_functions import display
import pandas as pd

# Cargar dataset
path = "data/datos.csv"
df_raw = pd.read_csv(path, sep=",", header="infer")

# Crear copia de trabajo
df = df_raw.copy()

# Total de filas
total_filas = len(df)
display(Markdown(f"Total de registros: {total_filas}"))
display(Markdown(f"#### **Muestra del conjunto de datos**"))
display(df.sample(n=10, ignore_index=True))


Total de registros: 150000

#### **Muestra del conjunto de datos**

,student_id,age,gender,course,year,daily_study_hours,daily_sleep_hours,screen_time_hours,stress_level,anxiety_score,depression_score,academic_pressure_score,financial_stress_score,social_support_score,physical_activity_hours,sleep_quality,attendance_percentage,cgpa,internet_quality,burnout_level
0,99011681,23.0,Female,BSc,1st,2.0,8.4,1.8,Medium,4.0,9.0,10.0,4,5.0,1.1,Average,78.7,9.07,Average,High
1,57245696,18.0,Male,BSc,3rd,1.3,6.1,8.5,Low,9.0,10.0,7.0,8,3.0,0.7,Average,65.4,6.98,Average,High
2,26386157,23.0,Female,BTech,2nd,3.2,6.2,8.8,Medium,5.0,1.0,10.0,2,3.0,0.1,Poor,59.0,9.68,Poor,High
3,50019192,19.0,Other,BSc,2nd,5.1,5.9,11.4,Medium,7.0,5.0,8.0,1,1.0,0.5,Average,67.9,6.47,Poor,High
4,66019149,17.0,Other,MBA,1st,5.2,6.1,3.6,Low,6.0,6.0,9.0,10,10.0,1.5,Average,67.6,4.98,Good,Low
5,17619417,23.0,Female,BCA,4th,9.5,8.2,4.3,Low,4.0,6.0,8.0,7,7.0,1.3,Average,73.4,6.04,Poor,High
6,55249839,21.0,Other,BSc,2nd,3.4,7.3,3.3,Low,4.0,8.0,5.0,9,1.0,0.8,Average,92.7,8.08,Good,Medium
7,24725195,23.0,Other,MBA,3rd,5.5,6.7,10.6,High,10.0,10.0,9.0,10,2.0,0.9,Good,76.9,9.43,Good,High
8,83934059,18.0,Female,MCA,3rd,7.1,9.0,8.8,Low,10.0,1.0,1.0,7,8.0,1.9,Good,67.5,6.57,Good,Low
9,27484277,17.0,Male,BTech,3rd,7.0,6.0,1.5,Low,8.0,10.0,10.0,3,7.0,1.3,Good,74.5,4.21,Poor,Low


#### Información de las columnas del conjunto de datos

In [31]:
# Describir el dataset

info_df = pd.DataFrame({
    'Columna': df.columns,
    'No Nulos': df.count().values,
    'Tipo de Dato': df.dtypes.values
})

display(info_df)

,Columna,No Nulos,Tipo de Dato
0,student_id,150000,int64
1,age,149999,float64
2,gender,149999,str
3,course,149997,str
4,year,149998,str
5,daily_study_hours,149996,float64
6,daily_sleep_hours,149999,float64
7,screen_time_hours,150000,float64
8,stress_level,149997,str
9,anxiety_score,149998,float64


### 📙Dicionario de Datos

<table style="width: 1400px; margin-left: 0; margin-right: auto;">
    <thead style="background-color: #352cba80">
        <tr>
            <td><b>Columna</b></td>
            <td><b>Formato</b></td>
            <td><b>Descripción</b></td>
            <td><b>Ejemplos Reales</b></td>
        </tr>        
    </thead>
    <tbody>
        <tr>
            <td><b>student_id</b></td>
            <td>Entero</td>
            <td>Identificador único para cada estudiante.</td>
            <td>73698903, 36386454 (Rango: 10000510 - 99999584)</td>
        </tr>
        <tr>
            <td><b>age</b></td>
            <td>Flotante</td>
            <td>Edad del estudiante.</td>
            <td>17.0, 20.0, 25.0</td>
        </tr>
        <tr>
            <td><b>gender</b></td>
            <td>Texto</td>
            <td>Género del estudiante.</td>
            <td>Male, Female, Other</td>
        </tr>
        <tr>
            <td><b>course</b></td>
            <td>Texto</td>
            <td>
                Programa académico matriculado:<br>
                • <b>BTech:</b> Licenciatura en Tecnología<br>
                • <b>BSc:</b> Licenciatura en Ciencias<br>
                • <b>BCA:</b> Licenciatura en Aplicaciones Informáticas<br>
                • <b>BBA:</b> Licenciatura en Administración de Empresas<br>
                • <b>MCA:</b> Maestría en Aplicaciones Informáticas<br>
                • <b>MBA:</b> Maestría en Administración de Empresas
            </td>
            <td>BTech, BCA, BSc, MBA, MCA, BBA</td>
        </tr>
        <tr>
            <td><b>year</b></td>
            <td>Texto</td>
            <td>Año de estudio del estudiante.</td>
            <td>1st, 2nd, 3rd, 4th</td>
        </tr>
        <tr>
            <td><b>daily_study_hours</b></td>
            <td>Flotante</td>
            <td>Tiempo promedio diario de estudio.</td>
            <td>4.3, 1.4 (Rango: 1.0 - 10.0)</td>
        </tr>
        <tr>
            <td><b>daily_sleep_hours</b></td>
            <td>Flotante</td>
            <td>Duración promedio diaria de sueño.</td>
            <td>6.8, 4.7 (Rango: 4.0 - 9.0)</td>
        </tr>
        <tr>
            <td><b>screen_time_hours</b></td>
            <td>Flotante</td>
            <td>Tiempo diario de exposición a pantallas.</td>
            <td>6.1, 3.0 (Rango: 1.0 - 12.0)</td>
        </tr>
        <tr>
            <td><b>stress_level</b></td>
            <td>Texto</td>
            <td>Categoría de estrés autorreportada.</td>
            <td>High, Medium, Low</td>
        </tr>
        <tr>
            <td><b>anxiety_score</b></td>
            <td>Flotante</td>
            <td>Puntaje de ansiedad (escala 1–10).</td>
            <td>10.0, 2.0 (Rango: 1.0 - 10.0)</td>
        </tr>
        <tr>
            <td><b>depression_score</b></td>
            <td>Flotante</td>
            <td>Puntaje de depresión (escala 1-10).</td>
            <td>3.0, 10.0 (Rango: 1.0 - 10.0)</td>
        </tr>
        <tr>
            <td><b>academic_pressure_score</b></td>
            <td>Flotante</td>
            <td>Puntaje de presión académica (escala 1-10).</td>
            <td>4.0, 8.0 (Rango: 1.0 - 10.0)</td>
        </tr>
        <tr>
            <td><b>financial_stress_score</b></td>
            <td>Entero</td>
            <td>Puntaje de estrés financiero (escala 1-10).</td>
            <td>2, 5 (Rango: 1 - 10)</td>
        </tr>
        <tr>
            <td><b>social_support_score</b></td>
            <td>Flotante</td>
            <td>Puntaje de apoyo social (escala 1-10).</td>
            <td>6.0, 9.0 (Rango: 1.0 - 10.0)</td>
        </tr>
        <tr>
            <td><b>physical_activity_hours</b></td>
            <td>Flotante</td>
            <td>Horas diarias de actividad física.</td>
            <td>1.8, 1.9 (Rango: 0.0 - 2.0)</td>
        </tr>
        <tr>
            <td><b>sleep_quality</b></td>
            <td>Texto</td>
            <td>Calidad del sueño reportada.</td>
            <td>Good, Average, Poor</td>
        </tr>
        <tr>
            <td><b>attendance_percentage</b></td>
            <td>Flotante</td>
            <td>Porcentaje de asistencia a clases.</td>
            <td>66.5, 55.8 (Rango: 50.0 - 100.0)</td>
        </tr>
        <tr>
            <td><b>cgpa</b></td>
            <td>Flotante</td>
            <td>Promedio de calificaciones acumulado (CGPA).</td>
            <td>9.63, 6.04 (Rango: 4.0 - 10.0)</td>
        </tr>
        <tr>
            <td><b>internet_quality</b></td>
            <td>Texto</td>
            <td>Calidad de la conexión a internet.</td>
            <td>Good, Average, Poor</td>
        </tr>
        <tr>
            <td><b>burnout_level</b></td>
            <td>Texto</td>
            <td>Nivel de agotamiento (burnout).</td>
            <td>High, Medium, Low</td>
        </tr>
    </tbody>
</table>